recipe/run_test.py

```diff
@@ -73,4 +73,4 @@
 import scipy.special
 
 
-sys.exit(not scipy.test(verbose=2))
+sys.exit(not scipy.test(verbose=2, extra_argv['-n4']))
```

Comment:
@hmaarrfk, how did you settle on `-n4` here?

Reply:
This is going to run with 4 threads on all platforms. Idk if this is an issue.

Reply:
Use CPU_COUNT env var?

Reply:
Yeah that's one thought. Though IDK what the level of parallelism is on Drone (if any) or if that level of parallelism is enough for it to complete.

Reply:
CPU count can be way too high for the amount of RAM  on certain computers. Shippable had 96 cores... Best to just use a reasonable number. Most CIs give two cores, so over prescribing by a factor of 2 isn't so bad especially with hyoerthreading